udata/core/dataset/models.py

```diff
@@ -590,7 +590,14 @@ def last_update(self):
             for res in self.resources:
                 dates += [res.modified, res.published]
                 if res.harvest and res.harvest.modified_at:
-                    dates.append(res.harvest.modified_at)
+                    if isinstance(res.harvest.modified_at, date):
```

Comment:
Could we move this to a small helper? `last_update` now seems very complicated, when it isn't :p